src/Parser/Monk/Windwalker/Modules/Spells/StormEarthAndFire.js

```diff
@@ -36,6 +46,57 @@ class StormEarthAndFire extends Analyzer{
     const reducedCooldownWithTraits = 90 - this.traitsCDReduction;
     return reducedCooldownWithTraits;
   }
+
+  on_toPlayer_applybuff(event) {
+    const spellId = event.ability.guid;
+    if (SPELLS.STORM_EARTH_AND_FIRE_CAST.id === spellId || SPELLS.SERENITY_TALENT.id === spellId) {
+      this.sefCount += 1;
```

Comment:
You could use the `AbilityTracker` module to get the number of casts instead of counting them yourself.